modules/tensor_mechanics/test/tests/beam/dynamic/tests

```diff
@@ -0,0 +1,74 @@
+    exodiff = 'dyn_euler_small_out.e'
+  [../]
+  [./dyn_timoshenko]
+    type = 'Exodiff'
+    input = 'dyn_timoshenko_small.i'
+    exodiff = 'dyn_timoshenko_small_out.e'
+  [../]
+  [./dyn_euler_rayleigh_hht]
+    type = 'Exodiff'
+    input = 'dyn_euler_small_rayleigh_hht.i'
+    exodiff = 'dyn_euler_small_rayleigh_hht_out.e'
+  [../]
+  [./dyn_euler_added_mass]
+    type = 'Exodiff'
+    input = 'dyn_euler_small_added_mass.i'
+    exodiff = 'dyn_euler_small_added_mass_out.e'
+  [../]
+  [./dyn_euler_added_mass_inertia_damping]
+    type = 'Exodiff'
+    input = 'dyn_euler_small_added_mass_inertia_damping.i'
+    exodiff = 'dyn_euler_small_added_mass_inertia_damping_out.e'
+  [../]
+  [./dyn_euler_added_mass2]
+    type = 'CSVDiff'
+    input = 'dyn_euler_small_added_mass2.i'
+    csvdiff = 'dyn_euler_small_added_mass2_out.csv'
+  [../]
+  [./error_1]
+    type = 'RunException'
+    input = 'dyn_euler_small.i'
+    cli_args = "Kernels/inertial_force_x/rotations='rot_x rot_y'"
+    expect_err = "InertialForceBeam: The number of variables supplied in 'displacements' and 'rotations' must match."
+    prereq = dyn_euler
+  [../]
+  [./error_2]
+    type = 'RunException'
+    input = 'dyn_euler_small_added_mass_inertia_damping.i'
+    cli_args = "NodalKernels/rot_x_inertial/rot_velocities='rot_vel_x rot_vel_y'"
+    expect_err = "NodalRotationalInertia: rot_velocities and rot_accelerations should be same size as rotations."
+    prereq = dyn_euler_added_mass_inertia_damping
+  [../]
+  [./error_3]
+    type = 'RunException'
+    input = 'dyn_euler_small_added_mass_inertia_damping.i'
+    cli_args = "NodalKernels/rot_x_inertial/Ixx=1 NodalKernels/rot_x_inertial/Iyy=1 NodalKernels/rot_x_inertial/Izz=1 NodalKernels/rot_x_inertial/Ixy=1"
+    expect_err = "NodalRotationalInertia: The moment of inertia tensor should be positive definite."
+    prereq = error_2
+  [../]
+  [./error_4]
+    type = 'RunException'
+    input = 'dyn_euler_small_added_mass_inertia_damping.i'
+    cli_args = "NodalKernels/rot_x_inertial/x_orientation='2 0 0' NodalKernels/rot_x_inertial/y_orientation='0 1 0'"
+    expect_err = "NodalRotationalInertia: x_orientation and y_orientation must be unit vectors."
+    prereq = error_3
+  [../]
+  [./error_5]
+    type = 'RunException'
+    input = 'dyn_euler_small_added_mass_inertia_damping.i'
+    cli_args = "NodalKernels/rot_x_inertial/x_orientation='1 0 0' NodalKernels/rot_x_inertial/y_orientation='1 0 0'"
+    expect_err = "NodalRotationalInertia: x_orientation and y_orientation should be perpendicular to each other."
+    prereq = error_4
+  [../]
+  [./error_6]
+    type = 'RunException'
+    input = 'dyn_euler_small_added_mass_inertia_damping.i'
+    cli_args = "NodalKernels/rot_x_inertial/x_orientation='1 0 0'"
+    expect_err = "NodalRotationalInertia: Both x_orientation and y_orientation should be provided if x_orientation or y_orientation is different from global x or y direction, respectively."
+    prereq = error_5
```

Comment:
Minor: Rather than serializing all of these error cases, you can always just turn off output instead and then they can run concurrently. This helps keep our testing system as fast and efficient as possible. You don't necessarily need to change this now, but you might think about that in the future.

Reply:
Thanks for the comment, @permcody. I have turned off csv and exodus output from the error tests and removed the prereq.